sunxspex/io.py

```diff
@@ -8,23 +8,16 @@
 import scipy.io
 from sunpy.io.special.genx import read_genx
 from sunpy.time import parse_time
+from sunpy.data import manager
 
-SSWDB_XRAY_CHIANTI = os.path.expanduser(os.path.join("~", "ssw", "packages",
-                                                     "xray", "dbase", "chianti"))
 
-
-def chianti_kev_line_common_load_light(linefile=None):
+def chianti_kev_line_common_load_light():
```

Comment:
If the user has there own local chianti_lines file, can they still read it somehow using this function?

Reply:
I think it would be using the same as above with `sunpy.data.manager` no?